lib/msal-common/src/account/IAccount.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export interface IAccount {
```

Comment:
This seems be used as a type, instead of an interface (you are constructing instances of it). I would prefer to see a concrete type that implements this interface, or change this to a type (and rename to `Account` or something like that.

Reply:
Changed in a future PR, will be up shortly.